consensus/src/round_manager.rs

```diff
@@ -920,6 +942,69 @@ impl RoundManager {
         Ok(vote)
     }
 
+    async fn process_order_vote_msg(&mut self, order_vote: OrderVote) -> anyhow::Result<()> {
+        if self.onchain_config.order_vote_enabled() {
+            fail_point!("consensus::process_order_vote_msg", |_| {
+                Err(anyhow::anyhow!("Injected error in process_order_vote_msg"))
+            });
+            info!(self.new_log(LogEvent::ReceiveOrderVote), "{}", order_vote);
+
+            if self
+                .pending_order_votes
+                .has_enough_order_votes(order_vote.ledger_info())
+            {
+                return Ok(());
+            }
+
+            if order_vote.ledger_info().round()
+                > self.block_store.sync_info().highest_commit_round()
```

Comment:
we only care about highest ordered round, not commit round?

Reply:
Done.